src/tools/wallmap/objloader.go

```diff
@@ -0,0 +1,194 @@
+package main
```

Comment:
Oh yes. Actually that's what I'd like to do, think of it as a micro
package. For example in the future more line parsing functions might be
added in order to support bigger set of the OBJ standard features.

Le 24 août 2016 01:25, "Giovanni Bajo" <notifications@github.com> a écrit :

> In src/tools/wallmap/objloader.go
> <https://github.com/RookieGameDevs/surviveler/pull/3#discussion_r75970292>
> :
>
> > @@ -0,0 +1,194 @@
> > +package main
>
> You're being very liberal with usage of exported names. Even if it's just
> a file in the main package, you're making things harder if you evolve this
> into a small package later. I'd suggest to still think of it as a
> micro-package and use lowercase names for everything but main entry points.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/RookieGameDevs/surviveler/pull/3/files/1f37e8cab422f8d46587682b13db9c63d44b6718#r75970292>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAdF6iafYww5BqaSHQM_ctUTYjZme4Itks5qi4FQgaJpZM4JrcmN>
> .
>